docs/contributing-to-registry.md

```diff
@@ -1,100 +0,0 @@
-> **Note**: The doc it's a work in progress and is subject to change. We are still creating the Bicep public registry, and the tool mentioned in the doc has not been published yet.
```

Comment:
The doc is moved to https://github.com/Azure/bicep-registry-modules/blob/main/CONTRIBUTING.md.